com.ibm.streamsx.topology/opt/python/packages/streamsx/rest.py

```diff
@@ -1,43 +1,43 @@
 # Licensed Materials - Property of IBM
 # Copyright IBM Corp. 2016,2017
-import requests
 import os
 import json
 import logging
-import streamsx.st as st
-
-from .rest_primitives import Domain, Instance, Installation, Resource, _StreamsRestClient, StreamingAnalyticsService,  _exact_resource
-from .rest_errors import ViewNotFoundError
 from pprint import pformat
 
+import streamsx.st as st
+from .rest_primitives import Domain, Instance, Installation, Resource, _StreamsRestClient, StreamingAnalyticsService, \
+    _exact_resource
+
 logger = logging.getLogger('streamsx.rest')
 
 
 class StreamsConnection:
-    """Creates a connection to a running Streams instance and exposes methods to retrieve the state of that instance.
+    """Creates a connection to a running distributed IBM Streams instance and exposes methods to retrieve the state of
+    that instance.
 
     Streams maintains information regarding the state of its resources. For example, these resources could include the
-    currently running Jobs, Views, PEs, Operators, and Domains. The StreamsConnection provides methods to retrieve that
-    information.
-
-    ``StreamsConnection`` connects to  distributed IBM Streams instance.
+    currently running Jobs, Views, PEs, Operators, and Domains. The :class:`StreamsConnection` provides methods to
```

Comment:
Ya, it did work.  But I didn't read anything in Sphinx documentation saying that :py is the default domain, so maybe it's better to put it in?